.github/workflows/main.yml

```diff
@@ -24,11 +24,30 @@ jobs:
     - name: Check Format (*.cs)
       run: dotnet format --verify-no-changes --verbosity diagnostic
 
+  Build-Test-Neo-Cli:
+    needs: [Format]
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+        
     - name: Build (Neo.CLI)
-      run: |
-        dotnet build ./src/Neo.CLI \
-        --output ./out/Neo.CLI
-
+      run: dotnet build ./src/Neo.CLI --output ./out/Neo.CLI
+        
+    - name: Install dependencies
+      run: |
+        sudo apt-get install libleveldb-dev expect
```

Comment:
leveldb is already installed?

Reply:
It was being installed below. I moved everything up and ensured dependency between tasks.

Reply:
In fact,I will need to revert because each task need their dep